### PR TITLE
keybase_service_base: call new service RPC to get ALL revoked keys

### PR DESCRIPTION
**Pull request by strib**
We previously had no way of getting all the pre-reset keys of a user, which led to situations where team folders are unreadable when they are last written by a user who then reset their account.

The service started exposing a V2 RPC that gives us all past incarnations of the user, so we can check all their keys, using the "reset" time as the revocation time.

Issue: KBFS-2842

### CODE REVIEW

File: libkbfs/keybase_daemon_rpc_test.go [hunk 1]
```diff
@@ -103,18 +103,21 @@ func (c *fakeKeybaseClient) Call(ctx context.Context, s string, args interface{}
 		c.identifyCalled = true
 		return nil
 
-	case "keybase.1.user.loadUserPlusKeys":
```
[the remark this answers is not in the document]
Comment (strib) [reply libkbfs/keybase_daemon_rpc_test.go]: Not sure, it was a core decision, but I guess it's probably easier than managing a new directory, copying over all the unchanged data types, etc.

**Comment by strib** [comment]
> Nice! Are there any performance concerns here now that we have more keys in the UserInfo?

I don't think so -- the revoked keys are in maps and I don't think there's any iterating in the happy path.  More keys to keep in memory I guess, but the service is already doing it so I don't think we're making things worse overall...

File: libkbfs/keybase_daemon_rpc_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jzila) [reply libkbfs/keybase_daemon_rpc_test.go]: Well I figured we'd just import `keybase.1` as appropriate. But fair enough. I guess `keybase.1` forever!

[the remark this answers is not in the document]
Comment (strib) [reply libkbfs/keybase_daemon_rpc_test.go]: Maybe we could also import `keybase.1`, but I was more thinking about what would happen if the `keybase.2` RPCs still referenced `keybase.1` data structures in the argument or response types, which they would probably have to...

**Pull request merged**